{"uuid": "d47a8d1f-e87c-4eb8-9482-4672684288a9", "children": ["2c1c0c77-2287-47ab-b8d9-bc10e3bcd78d"], "befores": [{"name": "remote", "status": "passed", "attachments": [{"source": "7ee6a023-3ba3-436f-a788-d140cfd494c2-attachment.json", "type": "application/json"}], "start": 1604856778743, "stop": 1604856780503}], "afters": [{"name": "remote::fin", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/usr/local/lib/python3.6/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/app/conftest.py\", line 52, in fin\n    driver.quit()\n  File \"/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 698, in quit\n    self.execute(Command.QUIT)\n  File \"/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/remote_connection.py\", line 402, in _request\n    resp = http.request(method, url, body=body, headers=headers)\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/request.py\", line 76, in request\n    method, url, fields=fields, headers=headers, **urlopen_kw\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/request.py\", line 97, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/poolmanager.py\", line 336, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 677, in urlopen\n    chunked=chunked,\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 426, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 421, in _make_request\n    httplib_response = conn.getresponse()\n  File \"/usr/local/lib/python3.6/http/client.py\", line 1379, in getresponse\n    response.begin()\n  File \"/usr/local/lib/python3.6/http/client.py\", line 311, in begin\n    version, status, reason = self._read_status()\n  File \"/usr/local/lib/python3.6/http/client.py\", line 272, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"/usr/local/lib/python3.6/socket.py\", line 586, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1604856783275, "stop": 1604856783587}], "start": 1604856778743, "stop": 1604856783587}